### ISSUE TITLE
Server Fleet Management Software

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#7]
## User story
1. As a SD
2. I want 
3. So that {}

## Acceptance criteria
* Decide on intergration pipeline

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
